Dockerfile

```diff
@@ -113,6 +113,8 @@ COPY --from=cfgov-build --chown=apache:apache ${CFGOV_PATH}/static.in ${CFGOV_PA
 RUN yum clean all && rm -rf /var/cache/yum && \
     chown -R apache:apache ${APP_HOME} ${SCL_HTTPD_ROOT}/usr/share/httpd ${SCL_HTTPD_ROOT}/var/run
 
+ENV PATH="/opt/rh/${SCL_PYTHON_VERSION}/root/usr/bin:${PATH}"
```

Comment:
This now lets the apache user run the django commands

Reply:
I'm a little squeamish about this, mainly because I think the apache user should have a few privileges as possible to lower attack surface area. Could these commands be run as a different user as part of the docker build? (this would likely require reorganizing the dockerfile a bit)

Reply:
not in its current form, and the apache user has access to python 2.7 as is so I figured the overall surface area for this to be somewhat low

Reply:
The reason behind why I ended up here, was the django command we need to run to seed the index relies on python 3.6 and the dependencies we install as part of the `cfgov-dev` stages. These are all installed as root. We could look to refine how we currently populate our elasticsearch to not rely on the python container and docker networking to reach elasticsearch and instead move these out to either a separate jenkins job or a separate stage executed by another container. The limiting factor there is we would need to expose the ES container via DNS.

Reply:
Overall, I think expanding the PATH variable for the apache user isn't too risky, as anyone capable of getting into a running container would have the ability to expand it anyways. Now if we want to evaluate trimming down our docker container even more then I think that is a good discussion to have.

Reply:
Yeah, I agree that this change itself is low risk and mirrors our conception of how the prod server should be set up and accessed. I suppose my point is that in a Docker regime, that conception is a bit flawed and we should try to move towards having running containers with minimal access rights/ability to run code other than what is their express purpose and to separate adjacent tasks into build stages/jobs as you suggest. Imo in docker, we should never expect to attach to the running prod container or use it to marshal tasks other than simply serving the website.